main.js

```diff
@@ -115,4 +115,67 @@ form.addEventListener('submit', (e) => {
   } else {
     errorEmail.style.display = 'none';
   }
-});
\ No newline at end of file
+});
+
+// Storage
+function storageAvailable(type) {
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return e instanceof DOMException && (
+    // everything except Firefox
+      e.code === 22
+            // Firefox
+            || e.code === 1014
+            // test name field too, because code might not be present
+            // everything except Firefox
+            || e.name === 'QuotaExceededError'
+            // Firefox
+            || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
+            // acknowledge QuotaExceededError only if there's something already stored
+            && (storage && storage.length !== 0);
+  }
+}
+
+// Storge
```

Comment:
[Optional] Kindly remove commented code in your project, so it looks clean and professional ✊

Please check for other occurrences. 🙂